src/rust/cryptography-x509-validation/src/ops.rs

```diff
@@ -45,10 +45,18 @@ pub(crate) mod tests {
         }
     }
 
-    #[test]
-    fn test_nullops() {
-        // Arbitrary relatively small cert (v1_cert.pem from cryptography_vectors).
-        let v1_cert = "
+    #[macro_export]
+    macro_rules! cert {
+        ($pem:literal) => {{
+            let parsed = Box::leak(Box::new(pem::parse($pem).unwrap()));
```

Comment:
I have questions. Namely: Why?

Reply:
Ah, this was a convenience macro while I was writing path validation tests. It's pretty ugly though and there's no repetition here, so I can just drop it.

Reply:
I don't mind a convenience macro, but there's no need to be leaking things :-)

Reply:
I'll see if I can work around it, but I _think_ the leak is necessary to get the lifetimes to work -- either a macro or a full function here will have local ownership of the `Pem` that's being borrowed from, so there's no way to get a lifetime shorter than `'static`.

Reply:
Okay, I've rewritten this as a fixture that returns a `pem::Pem` and a `cert(...)` helper -- that avoids the need to a leak, at the cost of a more explicit lifetime relationship between the two + binding for the `pem::Pem` at each callsite 🙂